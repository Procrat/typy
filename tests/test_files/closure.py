def f():
    x = 5
    def g():
        x + 1
        return x
    return g


f()()


def g():
    x = 5
    def h():
        x.lolcatz
        return x
    return h


g()()
