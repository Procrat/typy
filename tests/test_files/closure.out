Int has no attribute lolcatz
